srcpkgs/paleta/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'paleta'
+pkgname=paleta
+version=1.0.2
+revision=1
+build_style=gnu-makefile
+hostmakedepends="pkg-config tar"
```

Comment:
You need `tar` in `hostmakedepends`?

Reply:
I believe so since it has to decompress the tarball? Unless the xbps-src system does this automatically?

Reply:
When you specify a tar.gz archive in `distfiles`, it will fetch and extract it for you with `do_fetch()` and `do_extract()`. You don't need `tar` in `hostmakedepends` here

Reply:
Looking at the project, I don't think it uses `pkg-config`, though I haven't tested. I don't think you need anything in `hostmakedepends`.